.github/workflows/build-test.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       run: |
         export DISPLAY=:99
         sudo Xvfb -ac :99 -screen 0 1280x1024x24 > /dev/null 2>&1 &
+        echo 0| sudo tee /proc/sys/kernel/yama/ptrace_scope
```

Comment:
it is working.. there was a problem in test case I believe.. we should first terminate the launch and it will kill the applications we launched and then check if there is still some application running, if yes, destroy it..